docs/about/whats-new.ipynb

```diff
@@ -1087,7 +1087,7 @@
     "\n",
     "**New in 0.14**\n",
     "    \n",
-    "- [scipp.ndimage](../generated/modules/scipp.ndimage.rst) providing `gaussian_filter`, `median_filter`, and more.\n",
+    "- [scipp.scipy.ndimage](../generated/modules/scipp.scipy.ndimage.rst) providing `gaussian_filter`, `median_filter`, and more.\n",
```

Comment:
Should we remove these entries rather than make it look like the current module structure was introduced already in 0.14?

Reply:
Removed.